components/connect/components/search/view.coffee

```diff
@@ -15,8 +15,14 @@ module.exports = class ConnectSearchView extends Backbone.View
     input: ->
       @query arguments...
 
-  initialize: ({ @state, @search }) ->
+  initialize: ({ @state, @search, @eventBus }) ->
     @query = throttle @perform, 200
+    
+    @eventBus.on 'connection:added', @afterConnection
+
+  afterConnection: =>
+    @$el.focus()
+    @$el.select()
```

Comment:
> The advantage of using this form, instead of other.on(event, callback, object), is that listenTo allows the object to keep track of the events, and they can be removed all at once later on. The callback will always be called with object as context.